buildSrc/src/main/groovy/io.micronaut.build.internal.control-panel-module.gradle

```diff
@@ -2,17 +2,22 @@ plugins {
     id "io.micronaut.build.internal.module"
     id("org.sonatype.gradle.plugins.scan")
 }
+
 String ossIndexUsername = System.getenv("OSS_INDEX_USERNAME") ?: project.properties["ossIndexUsername"]
 String ossIndexPassword = System.getenv("OSS_INDEX_PASSWORD") ?: project.properties["ossIndexPassword"]
 boolean sonatypePluginConfigured = ossIndexUsername != null && ossIndexPassword != null
 ossIndexAudit {
     excludeCoordinates = ["com.google.guava:guava:18.0"]
-if (sonatypePluginConfigured) {
-    username = ossIndexUsername
-    password = ossIndexPassword
-}
+    if (sonatypePluginConfigured) {
+        username = ossIndexUsername
+        password = ossIndexPassword
+    }
```

Comment:
The whole `ossIndexAudit` block should be redundant when upgrading build plugins